samples/android/app/build.gradle

```diff
@@ -57,6 +59,9 @@ dependencies {
 ext.gleanNamespace = "mozilla.telemetry.glean"
 // Fixed build date so we can test for it
 ext.gleanBuildDate = "2020-11-06T11:30:50+00:00"
+// Enable expiration by major version. The sample is at version 1,
+// see versionName.
+ext.gleanExpireByVersion = 1
```

Comment:
Will we have some guidance how this value should be fetched from build variables at build time?

Reply:
well, if we want Fenix to use that we probably have to figure out how they can set it, don't we?